### DISCUSSION TITLE
metrics

### DISCUSSION THREAD

**Discussion by rhelopen** [signalfx/splunk-otel-collector-chart#719] [General] [answered]
Hello, Where do I define metrics token in values.yaml? I see only this for metrics.

  # Name of the Splunk metric type index targeted. Required when ingesting metrics to Splunk Platform.
  metricsIndex: "openshift_nonprod_metrics"

**Comment by matthewmodestino** [comment, marked as answer]
you do not need a separate token for metrics. 

A HEC Token is simply a transport auth mechanism. it does not need to be closely tied to what data is being sent. 

The collector will set the index in the payload and Splunk will receive it and route to the appropriate index.
